sheet_to_triples/run.py

```diff
@@ -50,7 +50,8 @@ def __init__(
     @classmethod
     def from_args(cls, args):
         if args.book:
-            book = {os.path.basename(b): xl.load_book(b) for b in args.book}
+            book = {os.path.basename(b): xl.load_book(b) for b in args.book
+                    if not os.path.basename(b).startswith('~$')}
```

Comment:
This seems like some weird platform-specific thing, so is likely better put behind a helper function that can have a descriptive name and specific logic, eg
```py
book = {os.path.basename(b): xl.load_book(b) for b in args.book if is_good_file(b)}
```
Then that function could also for instance, stat the path and check `FILE_ATTRIBUTE_HIDDEN` and such.